docs/config-file-curations-yml.md

```diff
@@ -69,6 +72,7 @@ The structure of the curations file consist of one or more `id` entries:
       revision: "1234abc"
       path: "subdirectory"
     is_meta_data_only: true
+    is_modified: true
```

Comment:
Out-of-scope and just something which came to my mind.
I thought I'd share it quickly and if more people like it we can make a ticket out of it.

Licenses may apply only to specific files of the package. E.g. if you include ' https://github.com/alex/certifi/blob/master/certifi/cacert.pem' in your project, wouldn't one then want to know whether the files which are under 'MPL' are modified as opposed to whether any file in the package has been modified. 

It could well be that package granularity for _is_modified_ and disregarding of the corresponding license of the modified files is not sufficient to implement real world use cases.

Reply:
I agree with you here. `is_modified` doesn't hold a lot of information about what has been modified and where. An improvement / additional feature could be useful here.